go/keybase/main.go

```diff
@@ -118,7 +118,9 @@ func mainInner(g *libkb.GlobalContext) error {
 	// This sends the client's PATH to the service so the service can update
 	// its PATH if necessary. This is called after FixVersionClash(), which
 	// happens above in configureProcesses().
-	if err = configurePath(g, cl); err != nil {
+	fc := cl.GetForkCmd()
+	err = configurePath(g, cl)
+	if err != nil && fc != libcmdline.NoFork {
```

Comment:
I changed main so that it won't error out of `configurePath` errors out. 

Is the point that `configurePath` should not happen on basic commands that don't require daemon access?  Or also not in standalone, right?

I think `configurePath` should move into `configureProcess` after `FixVersionClash`.